doc/OnlineDocs/modeling_extensions/gdp.rst

```diff
@@ -61,32 +60,36 @@ the example returns.
 Transformation
 --------------
 
-In order to use the solvers currently available, one must convert the
-disjunctive model to a standard MIP/MINLP model.  The easiest way to do
-that is using the (included) BigM or Convex Hull transformations.  From
-the Pyomo command line, include the option ``--transform
-pyomo.gdp.bigm`` or ``--transform pyomo.gdp.chull``
+To use standard commercial solvers, you must convert the disjunctive model to a standard MIP/MINLP model.
+The two classical strategies for doing so are the (included) Big-M and Hull reformulations.
```

Comment:
Is there a reason you're calling it the Hull reformulation instead of Convex Hull? The transformation is invoked using `chull` which might be a little confusing if you just refer to it as the Hull reformulation.

Reply:
Yes, because in literature it appears as the Hull Reformulation. Practically speaking, it is possible to apply it to nonconvex functions, although the theory around this is a bit sparse.

Reply:
This addresses an issue that might be a little imprecision from the notation in Pyomo. The reformulation is named the hull reformulation because it only takes the convex hull from the elements in the disjunction (if well written) but does not yield the actual object popularly known as convex hull from the object (i.e. if we forget about the integrality condition we wouldn't necessarily get an integer solution).

Reply:
I agree that the name may be misleading, but just to note: It is the true convex hull, but represented in a higher-dimensional space.

Reply:
Ok folks, I just checked with Ignacio yesterday. The Hull Reformulation is an exact reformulation from GDP into MINLP at the optimal solution, irregardless of convexity for the original GDP model constraints. The main complicating element of HR is the perspective function, but the function value is exact at indicator variable values of `y=1` and `y=0`. Therefore, its function value is exact for any feasible solutions of the GDP.

tldr: the perspective function of a nonconvex function will still be nonconvex, so the hull reformulation is still valid, but is not convex in the case of a GDP with nonconvex constraints.

Reply:
@qtothec so is this ready to merge?

Reply:
@blnicho Let me make a couple extra edits. I'll rerequest your review when I'm done.